build/testBuildImages.sh

```diff
@@ -6,6 +6,11 @@
 
 set -e
 
+echo
```

Comment:
Temporary code. We can remove it once we have identified the root cause of failed builds.